skimage/util/noise.py

```diff
@@ -27,7 +27,7 @@ def random_noise(image, mode='gaussian', seed=None, clip=True, **kwargs):
                       `low_val` is 0 for unsigned images or -1 for signed
                       images.
         - 'speckle'   Multiplicative noise using out = image + n*image, where
-                      n is uniform noise with specified mean & variance.
+                      n is gaussian noise with specified mean & variance.
```

Comment:
```suggestion
                      n is Gaussian noise with specified mean & variance.
```
so it matches https://github.com/scikit-image/scikit-image/blob/e19fa1f4c19f4df88a12307409efd8d2de1c4836/skimage/util/noise.py#L19-L20

Reply:
Sure, on it!